src/containers/BookingDetailModal/BookingDetailModal.container.jsx

```diff
@@ -198,28 +199,57 @@ const BookingDetailModalContainer = ({ bookingId, setNeededBooking }) => {
               )}`}
             />
           </div>
-          <Space>
-            <Typography>Give your feedback for this clinic!!</Typography>
-            <Button
-              type="primary"
-              onClick={() => {
-                handleGiveFeedback(bookingId);
-              }}
-            >
-              Give feedback
-            </Button>
-          </Space>
+          <div style={{ display: "flex", justifyContent: "space-between" }}>
```

Comment:
should we separate this into 2 file like "CSButtonActionGroup" and "PatientButtonActionGroup" ?